src/Shared/Shared.fsproj

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="Shared.fs" />
+    <None Include="paket.references" />
   </ItemGroup>
```

Comment:
Template uses Content for this reference. Does not seem like that would have any implications.